scripts/help

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+echo "Description: Makefile to build, test and package Harvester
+Documentation: https://github.com/harvester/harvester/wiki#build--development
+Source: https://github.com/harvester/harvester/tree/master/scripts
+Maintainer: @harvester/maintainer
+
+Targets:"
+
+inlined_targets=$(grep -E '^[a-zA-Z_-]+:.*?## ' "${1}" | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "  \033[36m%-20s\033[0m %s\n", $1, $2}')
```

Comment:
Sorry, I forgot about this. We might be able to remove it if all targets are expected to call into shell scripts. Otherwise, we will have to print them, if non-empty.

Reply:
Removed. Thanks